### ISSUE TITLE
watchOS 2 beta

### ISSUE THREAD

**Issue by dkhamsing** [Dimillian/SwiftHN#35]
Layout looks broken in watchOS 2 beta :cry: 
Only shows comments and url (no title)

![img_0845](https://cloud.githubusercontent.com/assets/4723115/8416166/b10ba87e-1e58-11e5-9e9e-9cca0a14b790.JPG)

**Comment by fluidpixel** [comment]
It's not 'broken' it's how the current storyboard is organised out.

**Comment by Dimillian** [comment]
Yeah, I've made that app in like 2 hours, the UI is very ugly. You can fix it :p

**Comment by dkhamsing** [comment]
2 hours! :grin:

**Comment by fluidpixel** [comment]
How about this?

![simulator screen shot 15 jul 2015 23 55 47](https://cloud.githubusercontent.com/assets/733214/8702904/163e6230-2b4d-11e5-9395-0295a8d1a068.png)

https://github.com/fluidpixel/SwiftHN/commit/63462df81c47c9d88ad08a83d65494091f453b2f

**Comment by Dimillian** [comment]
Works on iOS 8 + 9 ?

**Comment by fluidpixel** [comment]
@Dimillian I've only run it on iOS9 with xcode7, I can't vouch for 8 at the moment sorry.
